xml/docu_styleguide.contributors.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE appendix
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<appendix xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="sec-contributors">
+ <title>Contributors</title>
+
+ <para>
+  The following persons have contributed to this guide:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    Stefan Knorr
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Dmitri Popov
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Tanja Roth
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Thomas Schraitle
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Sven Seeberg
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Alexandra Settle
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Frank Sundermeyer
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Christoph Wickert
+   </para>
+  </listitem>
+ </itemizedlist>
```

Comment:
I wonder what to do about Jana & Edith (& potentially the Novell people that we can't even name anymore that originated many of the ideas in the guide).

Reply:
This appendix was meant as an example for how to create a contributor's appendix from the respective GitHub repository, I just listed all the people I found on the GH page for the style guide. As Jana, Edith, Martina, and Rebecca contributed in times when we still used SVN, they are not listed on GH and thus do not appear here.